scripts/station_csv_to_json.py

```diff
@@ -5,8 +5,19 @@
 import csv
 import json
 import re
+import geopandas
+from shapely.geometry import Point
 
-with open('csv/Station_BC.csv', 'r') as csvfile:
+def fetch_ecodivision_name(lat: str, long: str, ecodivisions: geopandas.GeoDataFrame):
+    """ Returns the ecodivision name for a given lat/long coordinate """
+    station_coord = Point(float(long), float(lat))
+    for index, row in ecodivisions.iterrows(): #pylint: disable=redefined-outer-name, unused-variable
+        geom = row['geometry']
+        if station_coord.within(geom):
+            return row['CDVSNNM']
+    return None
+
+with open('app/csv/Station_BC.csv', 'r') as csvfile:
```

Comment:
Can we keep the csv folder one level down please?  So just have it csv/Station_BC.csv?